backend/manager/modules/common/src/main/java/org/ovirt/engine/core/common/FeatureSupported.java

```diff
@@ -36,6 +37,11 @@ public static boolean hotPlugCpu(Version version, ArchitectureType arch) {
         return supportedInConfig(ConfigValues.HotPlugCpuSupported, version, arch);
     }
 
+    public static boolean hotPlugCpu(Version version, ArchitectureType arch, CpuPinningPolicy cpuPinningPolicy) {
+        return supportedInConfig(ConfigValues.HotPlugCpuSupported, version, arch) &&
```

Comment:
ok, so if we're adding a new method here - how about calling the other one? (`return hotPlugCpu(version, arch) && ...`)